lint/linter.py

```diff
@@ -569,17 +587,6 @@ def remove(cls, vid):
 
             del persist.view_linters[vid]
 
-    @classmethod
-    def reload(cls):
-        """Assign new instances of linters to views."""
-        for vid, linters in persist.view_linters.items():
-            for linter in linters:
-                linter.clear()
-                persist.view_linters[vid].remove(linter)
-                linter_class = persist.linter_classes[linter.name]
-                linter = linter_class(linter.view, linter.syntax)
-                persist.view_linters[vid].add(linter)
-
     @classmethod
     def clear_all(cls):
```

Comment:
@kaste Can we remove this?

Reply:
You need to check the callers

Reply:
Better said should we clear `raw_errors` here instead?
It is only called after the syntax map changes. But I also need to go through all linter plugins to see whether they harbor any callers.

Reply:
I have left the `clear` statement in, even after renaming `raw_errors` to `errors`.